tv_apps/smart-home/js/home.js

```diff
@@ -657,12 +725,25 @@
       this._cardPicker.show();
     },
 
+    onFilterChanged: function (filterName) {
+      if (filterName === FilterManager.FILTERS.ALL.name) {
+        this.mode = '';
+      } else {
+        this.mode = 'filter';
+      }
+    },
+
+    onCardPickerShow: function() {
+      this.mode = 'cardPicker';
```

Comment:
Updated to to bind context menu to card-list and folder-list so this mode is eliminated.